external/llvm-project/mlir/include/mlir/Conversion/Passes.td

```diff
@@ -105,6 +105,12 @@ def ArithToAMDGPUConversionPass : Pass<"convert-arith-to-amdgpu"> {
   }];
 
   let dependentDialects = ["amdgpu::AMDGPUDialect", "vector::VectorDialect"];
+
+  let options = [
+    Option<"saturateFP8Truncf", "saturate-fp8-truncf", "bool",
+           /*default=*/"false",
```

Comment:
is the default "False" ?

Reply:
For MIGraphX we set it to true, if you notice the changes to `Pipelines.cpp`

I'm defaulting it to false upstream for back-compat